schema/data/deploy-data.sh

```diff
@@ -205,6 +217,9 @@ createdb
 if [[ " ${actions[*]} " =~ " deploySwrs " ]]; then
   deploySwrs
 fi
+if [[ " ${actions[*]} " =~ " deployMocks " ]]; then
+  deployMocks
+fi
```

Comment:
This does not appear to be used. `deployMocks` should be added to the `actions` array in the `while` loop above.
This script's documentation should be updated to explain when the mocks schema is deployed, or better, make it an explicit option (`--deploy-mocks-schema`).

Reply:
the dev data now relies on the mocks schema being there, but the mocks schema needs to grant access to roles that are deployed with the portal schema I believe

Reply:
I'm thinking that `deployMocks` should be called for both the dev and test data. Is the "test" data even used for our tests, or are we still relying on the dev data being deployed for our tests?

you are calling `deployMocks` from within the `deployDevData` function above, so these lines are not really needed if we're not adding a `--deploy-mocks-schema` option

Reply:
I made it a little more flexible: dev and test data need the mocks schema to be deployed, so the script enforces it, but it's also accessible as a separate target if needed